pool.go

```diff
@@ -15,6 +15,24 @@ var ErrPoolEmpty = errors.New("connection pool is empty")
 
 var errPoolFull = errors.New("connection pool is full")
 
+var blockingCmds = map[string]bool{
+	"WAIT": true,
+
+	// taken from https://github.com/joomcode/redispipe#limitations
+	"BLPOP":      true,
+	"BRPOP":      true,
+	"BRPOPLPUSH": true,
+
+	"BZPOPMIN": true,
+	"BZPOPMAX": true,
+
+	"XREAD":      true,
+	"XREADGROUP": true,
+
+	"SAVE":  true,
+	"WATCH": true,
```

Comment:
It is worthless to watch for WATCH itself. WATCH is usable only in conjunction with following MULTI+EXEC.

Reply:
Done